contrib/pkginstaller/scripts/postinstall

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+# append /Applications/podman/bin to $PATH
+[ -f "$HOME/.bash_profile" ] && echo 'PATH=/Applications/podman/bin:$PATH' >> "$HOME"/.bash_profile
+[ -f "$HOME/.zshenv" ] && echo 'PATH=/Applications/podman/bin:$PATH' >> "$HOME"/.zshenv
+[ -f "$HOME/.config/fish/config.fish" ] && echo 'set PATH /Applications/podman/bin $PATH' >> "$HOME"/.config/fish/config.fish
+
+# create containers.conf config file to set the location of helper binaries
+cat << EOF > "$HOME"/.config/containers/containers.conf
+[containers]
```

Comment:
Only write to a file when this doesn't exist. Otherwise an in-place change for the helper app is needed.

This currently can remove settings.

Reply:
have to look into in place updating the config file, is there a way to set it via the `podman cli` ?